README.md

```diff
@@ -56,6 +62,88 @@ If you have a Snyk token, add that as an environment variable:
 - Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables.
 - Set the variable name to _EXHORT_SNYK_TOKEN_, and copy-and-paste your Snyk token into the value field.
 
+
+#### General Configuration
+ Click on Manage Jenkins -> System, scroll down to Global properties/Environment Variables, And there you can set the following settings:
+ - name: _EXHORT_DEBUG_, Value: true , Description: Will invoke the analysis in verbose mode and will print a lot of useful logs to job output console - good for debugging, Default value is false.
+
+
+ - name: _EXHORT_DEV_MODE_, value: true, Description: Will invoke the Analysis on Staging Instance Of EXHORT Service, Default: false ( EXHORT Production Instance)
+
+
+ - name: _HIGHEST_ALLOWED_VULN_SEVERITY_, Possible values: [`LOW`,`MEDIUM`,`HIGH`,`CRITICAL`], Description: will determine what is the highest allowed Severity of a vulnerability found for a given package/dependency in the analysis, for the analysis to be considered Successful(RC=0) and not Vulnerable(RC=2), Default value is `MEDIUM`
+
+#### Python Pipeline Configuration
+ For python PIP Packages, you can bypass using the python and pip binaries during the invocation of the analysis, and just use them elsewhere in the pipeline job (another stage, or even another agent/node),
+ The Idea is to bring maximum flexibility with the python and pip versions, so you won't enforce the user to install sometimes a different python and pip versions just to adapt it to the exact requirements.txt' packages' versions, as python pip is very sensitive to versioning (for each python and pip version , there is only a limited range of supported versions for every package).
+ There are two environment variables:
+  1. _EXHORT_PIP_FREEZE_
+  2. _EXHORT_PIP_SHOW_ 
+ 
+
+ This feature enable you to use python on different agent ( for example, container image of python containing the desired python version), install the input requirements.txt using pip in the container image, and then invoke `pip freeze --all` and `pip show [List_Of_Packages]`, and save the output of these to a workspace files,
+ And then set the above variables with these output after encoding them using base64 encoding.
+
+Example Pipeline With proper usage:
```

Comment:
You only need to capitalize the first word.

`Example Pipeline With proper usage:` --> `Example pipeline with proper usage:`

Reply:
Done.